Update WormToggleAnimation.kt

m37moud opened the pull request:
feat: Update WormToggleAnimation.kt with RTL support

The WormToggleAnimation.kt file was updated to adjust coordinates for Right-to-Left (RTL) layout direction.

m37moud committed:
Update WormToggleAnimation.kt

feat: Update WormToggleAnimation.kt with RTL support

The WormToggleAnimation.kt file was updated to adjust coordinates for Right-to-Left (RTL) layout direction.

ilyasipek commented on app/src/main/java/com/ilyasipek/composeanimations101/animations/WormToggleAnimation.kt:
```diff
@@ -32,6 +32,9 @@ import com.ilyasipek.composeanimations101.ui.theme.ComposeAnimations101Theme
 import kotlinx.coroutines.launch
+import androidx.compose.ui.platform.LocalLayoutDirection
+import androidx.compose.ui.unit.LayoutDirection
+
```
Please remove the empty line

m37moud replied:
thanks for reviewing the line removed

ilyasipek commented on app/src/main/java/com/ilyasipek/composeanimations101/animations/WormToggleAnimation.kt:
```diff
@@ -94,9 +99,18 @@ fun WormToggleAnimation(
                 .padding(horizontalPadding)
                 .requiredSize(indicatorSize)
                 .drawBehind {
+               // Adjust coordinates for RTL:
```
Please format this line

m37moud replied:
format  is done

ilyasipek reviewed (changes requested):
Hi @m37moud, thanks for your PR, works great 🔥, I added 2 comments about styling, please fix them and I will merge it ASAP.

m37moud committed:
Update WormToggleAnimation.kt

Fix styling: remove extra empty line and format RTL comment per review feedback
